src/App.tsx

```diff
@@ -1,13 +1,22 @@
 import './App.css';
 import Frame from './components/frame';
+import Commodity from './components/comodities/comodities';
+import CurrencyExchange from './components/currency/currency';
 
 function App() {
   return (
     <div className='App'>
       {/* You can delete this... */}
       <Frame title='Title'>
         <p>This is a test</p>
+        <div>
+        {Commodity()}        
```

Comment:
so each of your components should be wrapped in their own frame. I kind of think we should do the frames on the component level to relieve clutter and conflicts

Reply:
Added frames on component level